waiter/src/waiter/process_request.clj

```diff
@@ -287,10 +287,18 @@
         request-body-streaming-counter (metrics/service-counter service-id "request-counts" "request-body-streaming")
         body-ch (async/chan 2048)
         report-request-size-metrics (let [bytes-streamed-atom (atom 0)
-                                          statsd-unreported-bytes-atom (atom 0)]
+                                          statsd-unreported-bytes-atom (atom 0)
+                                          throughput-meter (metrics/service-meter service-id "streaming" "request-bytes")
+                                          throughput-meter-global (metrics/waiter-meter "streaming" "request-bytes")
+                                          throughput-packets-meter (metrics/service-meter service-id "streaming" "request-packets")
+                                          throughput-packets-meter-global (metrics/waiter-meter "streaming" "request-packets")]
                                       (fn report-request-size-metrics [bytes-read complete?]
                                         (try
                                           (when (pos? bytes-read)
+                                            (meters/mark! throughput-meter bytes-read)
+                                            (meters/mark! throughput-meter-global bytes-read)
+                                            (meters/mark! throughput-packets-meter 1)
```

Comment:
Why do we expect that each iteration here is 1 packet?  Are we talking about 1 IP packet?

Reply:
1 is optional. Let's remove.

Reply:
> Why do we expect that each iteration here is 1 packet? Are we talking about 1 IP packet?

Not an IP packet, one buffered packet in Waiter code. It can help reflect inefficiency if there are many packets with small sizes.

> 1 is optional. Let's remove.

Removed.